integration/getServices/client_get_services_test.go

```diff
@@ -0,0 +1,74 @@
+package integration
```

Comment:
ok right package but wrong directory :) needs to move back one to the integration dir

Reply:
will do, any suggestions where the golden files should go, `integration/getServicesTestData`?